frontend/src/app/modules/intervention-form/form/form.component.scss

```diff
@@ -1,48 +1,53 @@
 .InterventionFormCard {
```

Comment:
Diff seems to be incorrectly parsed by git - most of the changes were automatic by Prettier I think

Reply:
Only real difference is adding `&--header` rules

Reply:
Yeah, the spacing was changed from 4 spaces to 2 spaces 👍